### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by chamathns**
### Proposed changes in this pull request

Update README.md to reflect SAML SDK configurations.

### When should this PR be merged

ASAP

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,420 @@
-# identity-agent-sso
\ No newline at end of file
+# WSO2 SAML SDK for Java
+
+The WSO2 Security Assertions Markup Language (SAML) Software Development Kit (SDK) for Java enables software
```
Comment (darshanasbg) [inline README.md:3]: Lets refer as `The WSO2 SAML SDK for Java` for simplysity.

File: README.md [hunk 2]
```diff
@@ -1 +1,420 @@
...
+ developers to integrate SAML based SSO authentication with Java Web applications. The SDK is built on top of the
+  OpenSAML library which allows Java developers to develop cross-domain single sign-on and federated access control
+   solutions with minimum hassle.
+
+## Getting Started
+
+These instructions will guide you on integrating SAML into your Java application with the WSO2 SAML SDK.
```
Comment (darshanasbg) [inline README.md:10]: Better start with 
### Trying out the sample
section.

File: README.md [hunk 3]
```diff
@@ -1 +1,420 @@
...
+- Single Sign-On (SSO) and Single Log-Out (SLO) (SP-Initiated and IdP-Initiated).
+- Assertion and nameID encryption.
+- Assertion signatures.
+- Message signatures: AuthNRequest, LogoutRequest, LogoutResponses.
+- Enable an Assertion Consumer Service endpoint.
+- Enable a Single Logout Service endpoint.
+- Publish the SP metadata.
+
+### Trying out the sample
+A sample app for demonstrating SAML based SSO authentication, SLO and attribute retrieval is hosted at:
```
Comment (darshanasbg) [inline README.md:23]: Trying out the sample better to be straight forward, without user to select through different options.

So let's instruct to download the war file from the given link.

File: README.md [hunk 4]
```diff
@@ -1 +1,420 @@
...
+https://github.com/wso2-extensions/identity-agent-sso/tree/master/resources/SampleApp
+
+You can either download the pre-built SampleApp.war from here[link] or build it from the source:
+
+#### Building from the source
```
Comment (darshanasbg) [inline README.md:28]: Let's have separate section on building the whole repo in a later section. Building specifically the sample from the source is not needed

Comment (ChamathNS) [reply README.md]: +1. Thanks. The comments have been addressed.

**Review by darshanasbg — APPROVED** [review]

**Pull request merged**